tests/test_chain.rs

```diff
@@ -1,7 +1,7 @@
 use miette::{miette, Report};
 
 fn error() -> Report {
-    miette!(0).wrap_err(1).wrap_err(2).wrap_err(3)
+    miette!("0").wrap_err(1).wrap_err(2).wrap_err(3)
```

Comment:
Ooh is this a breaking change?

Reply:
Well yes, but I doubt that someone uses miette like that

Reply:
I'll just pretend I didn't see it ;)